index.js

```diff
@@ -38,12 +41,15 @@ function Scroller(scroller, content, render, isPrepend, isSticky, cb) {
       var m = queue.shift()
       var r = render(m)
       append(scroller, content, r, isPrepend, isSticky)
```

Comment:
side note : I think `append` could be more clearly named something like `inject` or `insert` given append could potentially prepend